code/_onclick/hud/hud.dm

```diff
@@ -205,6 +205,83 @@ datum/hud/New(mob/owner)
 	else if(isguardian(mymob))
 		guardian_hud()
 
+	create_parallax()
+
+
+/client/var/list/obj/screen/spessbg/spessbg = list()
+
+var/list/client/parallax_on_clients = list()
+
+var/area/global_space_area = null
+
+/obj/screen/spessbg
+	var/offset_x = 0
+	var/offset_y = 0
+/datum/hud/var/showing_parallax
+/datum/hud/proc/create_parallax()
+	var/client/C = mymob.client
+	showing_parallax = C.prefs.space_parallax
+	if (C.prefs.space_parallax)
+		parallax_on_clients |= C
+	else
+		parallax_on_clients -= C
+	for(var/area/A in all_areas)
+		if(A.white_overlay)
+			if (C.prefs.space_parallax)
+				C.images |= A.white_overlay
+			else
+				C.images -= A.white_overlay
+	if (C.spessbg.len)
+		for(var/obj/screen/spessbg/bgobj in C.spessbg)
+			bgobj.layer = (C.prefs.space_parallax) ? AREA_LAYER + 0.5 : 0
+			C.screen |= bgobj
+		return
+	for(var/i=0;i<4;i++)
```

Comment:
for lists should have spacing between the operators, and use `,` to seperate the three parts; `for(var/i = 0, i < 4, i++)`